native_client/Makefile

```diff
@@ -26,9 +26,15 @@ CFLAGS      := -isystem $(RASPBIAN)/usr/include -L$(RASPBIAN)/lib/arm-linux-gnue
 LDFLAGS     := -Wl,-rpath-link,$(RASPBIAN)/lib/arm-linux-gnueabihf -Wl,-rpath-link,$(RASPBIAN)/usr/lib/arm-linux-gnueabihf/ -lsox
 endif
 
+OS := $(shell uname -s)
 CFLAGS  += -L${TFDIR}/bazel-bin/tensorflow -L${TFDIR}/bazel-bin/native_client $(EXTRA_CFLAGS)
 LDFLAGS += -ltensorflow -ldeepspeech -lc_speech_features -lkissfft $(EXTRA_LDFLAGS)
 
+META_LD_LIBRARY_PATH=LD_LIBRARY_PATH
```

Comment:
Would be nice for this to match the rest of the file sylistically, so `META_LD_LIBRARY_PATH := LD_LIBRARY_PATH`, etc.